packages/discord.js/src/util/APITypes.js

```diff
@@ -529,3 +529,8 @@
  * @external WebhookType
  * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/enum/WebhookType}
  */
+
+/**
+ * @external UserPremiumType
```

Comment:
Can you place this after `UserFlags` so the file stays alphabetically sorted (by `@external` tag)?